src/screens/ExchangeModal.tsx

```diff
@@ -82,13 +91,18 @@ export const DEFAULT_SLIPPAGE_BIPS = {
   [Network.polygon]: 200,
   [Network.optimism]: 200,
   [Network.arbitrum]: 200,
+  [Network.goerli]: 100,
```

Comment:
@estebanmino Ive added these test networks to be exhaustive as TS will complain if we dont